### PR TITLE
[wandb] Fix Confusion Matrix Plugin

### PR DESCRIPTION
**Pull request by Mathieu4141**
`BaseStrategy.logits` was renamed to `BaseStrategy.mb_outputs`, but wandb confusion matrix plugin got forgotten.

### CODE REVIEW

**Comment by vlomonaco** [comment]
Thanks @Mathieu4141 for your contribution and welcome to the avalanche family! :) @AndreaCossu will check this out and merge it!

**Comment by AndreaCossu** [comment]
Thanks @Mathieu4141 for the fix. I'll merge this PR as soon as tests are fixed. I don't know why are failing right now, so let me investigate into this. It may affect the other PR you submitted :smile:

**Comment by AndreaCossu** [comment]
Hi @Mathieu4141 tests should work now. Can you please make a new commit so that actions are triggered again?
You can use 
```
git commit --allow-empty -m "trigger GitHub actions"
git push
```

Please, do it for all your PRs :smile:

**Comment by Mathieu4141** [comment]
Thanks! 
This branch should be up to date now 😄

**Comment by AndreaCossu** [comment]
Tests are passing, I'm merging this! Thanks again :smile:
